src/bin/rgb.rs

```diff
@@ -382,11 +387,43 @@ fn main() -> Result<(), Error> {
             }
         },
         Command::Psbt { subcommand } => match subcommand {
-            PsbtCommand::Bundle { psbt_in, psbt_out } => {
+            PsbtCommand::Bundle {
+                psbt_in,
+                psbt_out,
+                method,
+            } => {
                 let psbt_bytes = fs::read(&psbt_in)?;
                 let mut psbt = Psbt::deserialize(&psbt_bytes)?;
 
-                let count = psbt.rgb_bundle_to_lnpbp4()?;
+                let mut count: usize = 0;
+                match method {
+                    CloseMethod::TapretFirst => {
+                        if let Some(output) =
+                            psbt.outputs.iter_mut().find(|o| o.script.is_v1_p2tr())
+                        {
+                            if output.tapret_dfs_path().is_none() {
+                                output
+                                    .set_tapret_dfs_path(&DfsPath::with([&DfsOrder::First]))
```

Comment:
I think it was my bad: it has to be
```suggestion
                                    .set_tapret_dfs_path(&DfsPath::with([&DfsOrder::Last]))
```

Reply:
could have spotted that while copying from the propose change, sorry. now fixed